public/app/plugins/panel/table/module.tsx

```diff
@@ -104,6 +104,70 @@ export const plugin = new PanelPlugin<Options, FieldConfig>(TablePanel)
           name: 'Hide in table',
           defaultValue: undefined,
           hideFromDefaults: true,
+        })
+        .addBooleanSwitch({
+          path: 'footer.show',
```

Comment:
this can't work at the field level. i think we can remove it and just auto-enable the footer if any fields have a reducer configured?